packages/tour-kit/src/components/tour-kit-step.tsx

```diff
@@ -15,20 +11,14 @@ const TourKitStep: React.FunctionComponent< Props > = ( {
 	config,
 	steps,
 	currentStepIndex,
+	classNames,
 	onMinimize,
 	onDismiss,
 	onNext,
 	onPrevious,
 	setInitialFocusedElement,
 	onGoToStep,
 } ) => {
-	const classNames = classnames(
```

Comment:
Let's maybe instead move `classSplit` above into a utils folder and reuse here to keep concerns isolated (so `step` owns its class definitions). Frame is owning a little too much as is already.

Reply:
Should I make a utils directory or just a utils file? Should the dir/file go in the components directory or should it go directly into src?

Let me know if we have some more stuff that you think can be pulled out I can add them in there too.

Reply:
Good question. Well for now, given the size of the package, we can keep in a single file, thinking under `src` (or a `src/utils/index.ts` one if you prefer). Maybe add a test to go along?

We can keep an eye out for reusable utilities and see if down the line we can split out.